src/locales/fr.ts

```diff
@@ -62,325 +62,293 @@ export default {
+    thomas_becket_bishop: 'Saint Thomas Becket, Évêque et Martyr (✝ 1170)',
+    thomas_apostle: 'Saint Thomas, Apôtre',
+    turibius_of_mogrovejo_bishop: 'Saint Alphonse Turibe de Mogrovejo, Évêque de Lima (✝ 1606)',
+    vincent_of_saragossa_deacon: 'Saint Vincent, Diacre et Martyr (✝ 304)',
+    vincent_de_paul_priest: 'Saint Vincent de Paul, Prêtre, Fondateur de la congrégation de la Mission et des Filles de la Charité (✝ 1660)',
+    vincent_ferrer_priest: 'Saint Vincent Ferrier, Prêtre de l’Ordre des Prêcheurs (✝ 1419)',
+    wenceslaus_i_of_bohemia_martyr: 'Saint Venceslas, Martyr (✝ 929)',
+    seven_holy_founders_of_the_servite_order: 'Saint Alexis Falconieri et les fondateurs des Servites (✝ 1310)',
+    sunday_of_the_word_of_god: 'Dimanche de la Parole de Dieu',
+    passion_of_saint_john_the_baptist: 'Martyre de Saint Jean-Baptiste',
+    most_holy_name_of_jesus: 'Saint Nom de Jésus',
+    visitation_of_mary: 'Visitation de la Vierge Marie',
+    our_lady_of_sorrows_principal_patroness_of_slovakia: 'Notre-Dame des Douleurs, Patronne Principale de la Slovaquie',
+    george_of_lydda_martyr_principal_patron_of_england: 'Saint Georges, Martyr et Patron Principal de l’Angleterre (✝ 303)',
+    benedict_of_nursia_abbot_principal_patron_of_europe: 'Saint Benoît de Nursie, Patriarche des moines d’Occident, Fondateur de l’ordre des Bénédictins et Patron Principal de l’Europe (✝ v. 547)',
+    cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop_secondary_patrons_of_europe: 'Saints Cyrille, Moine, et Méthode, Évêque, Apôtres des Slaves au 9ème siècle, Copatrons de l’Europe',
+    teresa_benedicta_of_the_cross_stein_virgin_secondary_patron_of_europe: 'Sainte Thérèse-Bénédicte de la Croix (Edith Stein), Carmélite, Martyr en Pologne, Copatronne de l’Europe (✝ 1942)',
+    andrew_apostle_principal_patron_of_scotland: 'Saint André, Apôtre et Patron Principal de l’Ecosse',
+    bridget_of_sweden_religious_secondary_patroness_of_europe: 'Sainte Brigitte de Suède, Veuve puis Religieuse, Fondatrice de l’ordre du Saint-Sauveur, Copatronne de l’Europe (✝ 1373)',
+    catherine_of_siena_virgin_secundary_patroness_of_europe: 'Sainte Catherine de Sienne, Vierge, Docteur de l’Eglise, Copatronne de l’Europe (✝ 1380)',
+    catherine_of_siena_virgin_secondary_patroness_of_italy_and_europe: 'Sainte Catherine de Sienne, Vierge, Docteur de l’Eglise, Copatronne de l’Italie et de l’Europe (✝ 1380)',
+    patrick_of_ireland_bishop_principal_patron_of_ireland: 'Saint Patrick, Évêque, Patron Principal de l’Irlande (✝ 461)',
+    columba_of_iona_abbot_secondary_patron_of_ireland: 'Saint Colomba, Religieux, Patron Secondaire de l’Irlande (✝ 615)',
+    our_lady_of_guadalupe_principal_patroness_of_the_americas: 'Notre-Dame de Guadalupe, Patronne Principale des Amériques',
+    francis_of_assisi_principal_patron_of_italy: 'Saint François d’Assise, Fondateur de l’ordre des Frères mineurs, Patron de l’Italie (✝ 1226)',
```

Comment:
@emagnier, could you proof the French patronage titles please? Below I list them all.

```js
joachim_and_anne_principal_patroness_of_the_province_of_quebec_parents_of_mary: 'Saints Anne, Patronne Principale du Québec, et Joachim, Parents de la Vierge Marie (1er siècle)',
joan_of_arc_virgin_secondary_patroness_of_france: 'Sainte Jeanne d’Arc, Vierge, Patronne Secondaires de la France (✝ 1431)',
joseph_spouse_of_mary_principal_patron_of_canada: 'Saint Joseph, Époux de la Bienheureuse Vierge Marie, Principal Patron du Canada',
john_de_brebeuf_isaac_jogues_priests_and_companions_martyrs_secondary_patrons_of_canada: 'Saints Jean de Brébeuf, Isaac Jogues, Prêtres Jésuites, et leurs compagnons, Martyrs, Seconds Patrons du Canada (17ème siècle)',
therese_of_the_child_jesus_and_the_holy_face_of_lisieux_virgin_secondary_patroness_of_france: 'Sainte Thérèse de l’Enfant-Jésus, Vierge, Docteur de l’Église, Seconde Patronne de la France (✝ 1897)',
our_lady_of_sorrows_principal_patroness_of_slovakia: 'Notre-Dame des Douleurs, Patronne Principale de la Slovaquie',
george_of_lydda_martyr_principal_patron_of_england: 'Saint Georges, Martyr et Patron Principal de l’Angleterre (✝ 303)',
benedict_of_nursia_abbot_principal_patron_of_europe: 'Saint Benoît de Nursie, Patriarche des moines d’Occident, Fondateur de l’ordre des Bénédictins et Patron Principal de l’Europe (✝ v. 547)',
cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop_secondary_patrons_of_europe: 'Saints Cyrille, Moine, et Méthode, Évêque, Apôtres des Slaves au 9ème siècle, Copatrons de l’Europe',
teresa_benedicta_of_the_cross_stein_virgin_secondary_patron_of_europe: 'Sainte Thérèse-Bénédicte de la Croix (Edith Stein), Carmélite, Martyr en Pologne, Copatronne de l’Europe (✝ 1942)',
andrew_apostle_principal_patron_of_scotland: 'Saint André, Apôtre et Patron Principal de l’Ecosse',
bridget_of_sweden_religious_secondary_patroness_of_europe: 'Sainte Brigitte de Suède, Veuve puis Religieuse, Fondatrice de l’ordre du Saint-Sauveur, Copatronne de l’Europe (✝ 1373)',
catherine_of_siena_virgin_secundary_patroness_of_europe: 'Sainte Catherine de Sienne, Vierge, Docteur de l’Eglise, Copatronne de l’Europe (✝ 1380)',
catherine_of_siena_virgin_secondary_patroness_of_italy_and_europe: 'Sainte Catherine de Sienne, Vierge, Docteur de l’Eglise, Copatronne de l’Italie et de l’Europe (✝ 1380)',
patrick_of_ireland_bishop_principal_patron_of_ireland: 'Saint Patrick, Évêque, Patron Principal de l’Irlande (✝ 461)',
columba_of_iona_abbot_secondary_patron_of_ireland: 'Saint Colomba, Religieux, Patron Secondaire de l’Irlande (✝ 615)',
our_lady_of_guadalupe_principal_patroness_of_the_americas: 'Notre-Dame de Guadalupe, Patronne Principale des Amériques',
francis_of_assisi_principal_patron_of_italy: 'Saint François d’Assise, Fondateur de l’ordre des Frères mineurs, Patron de l’Italie (✝ 1226)',
```

Reply:
Here is the corrections.
I notice that in french in the missal and lectionary, we don't specify the word "principal". We only keep "secondary" when it's needed. Also all titles (except proper names) should be lowercase.
There is also a lot of titles that should be better titled, according to the latest publishing of the sanctoral book. The new official missal in french should also be out within a month or 2. I will try to submit soon a future PR based on this new missal and the sanctoral book.

```yaml
joachim_and_anne_principal_patroness_of_the_province_of_quebec_parents_of_mary:
  'Saints Anne, Patronne de la province civile de Québec, et Joachim, Parents de la Vierge Marie (1er siècle)',
joan_of_arc_virgin_secondary_patroness_of_france:
  'Sainte Jeanne d’Arc, Vierge, Patronne Secondaire de la France (✝ 1431)',
joseph_spouse_of_mary_principal_patron_of_canada:
  'Saint Joseph, Époux de la Bienheureuse Vierge Marie, Patron du Canada',
john_de_brebeuf_isaac_jogues_priests_and_companions_martyrs_secondary_patrons_of_canada:
  'Saints Jean de Brébeuf, Isaac Jogues, Prêtres Jésuites, et leurs compagnons, Martyrs, Seconds Patrons du Canada (17ème siècle)',
therese_of_the_child_jesus_and_the_holy_face_of_lisieux_virgin_secondary_patroness_of_france:
  'Sainte Thérèse de l’Enfant-Jésus, Vierge, Docteur de l’Église, Seconde Patronne de la France (✝ 1897)',
our_lady_of_sorrows_principal_patroness_of_slovakia: 'Notre-Dame des Douleurs, Patronne de la Slovaquie',
george_of_lydda_martyr_principal_patron_of_england: 'Saint Georges, Martyr et Patron de l’Angleterre (✝ 303)',
benedict_of_nursia_abbot_principal_patron_of_europe:
  'Saint Benoît de Nursie, Patriarche des moines d’Occident, Fondateur de l’ordre des Bénédictins et Patron de l’Europe (✝ v. 547)',
cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop_secondary_patrons_of_europe:
  'Saints Cyrille, Moine, et Méthode, Évêque, Apôtres des Slaves au 9ème siècle, Copatrons de l’Europe',
teresa_benedicta_of_the_cross_stein_virgin_secondary_patron_of_europe:
  'Sainte Thérèse-Bénédicte de la Croix (Edith Stein), Carmélite, Martyr en Pologne, Copatronne de l’Europe (✝ 1942)',
andrew_apostle_principal_patron_of_scotland: 'Saint André, Apôtre et Patron de l’Écosse',
bridget_of_sweden_religious_secondary_patroness_of_europe:
  'Sainte Brigitte de Suède, Veuve puis Religieuse, Fondatrice de l’ordre du Saint-Sauveur, Copatronne de l’Europe (✝ 1373)',
catherine_of_siena_virgin_secundary_patroness_of_europe:
  'Sainte Catherine de Sienne, Vierge, Docteur de l’Eglise, Copatronne de l’Europe (✝ 1380)',
catherine_of_siena_virgin_secondary_patroness_of_italy_and_europe:
  'Sainte Catherine de Sienne, Vierge, Docteur de l’Eglise, Copatronne de l’Italie et de l’Europe (✝ 1380)',
patrick_of_ireland_bishop_principal_patron_of_ireland: 'Saint Patrick, Évêque, Patron de l’Irlande (✝ 461)',
columba_of_iona_abbot_secondary_patron_of_ireland:
  'Saint Colomba, Religieux, Patron Secondaire de l’Irlande (✝ 615)',
our_lady_of_guadalupe_principal_patroness_of_the_americas: 'Notre-Dame de Guadalupe, Patronne des Amériques',
francis_of_assisi_principal_patron_of_italy:
  'Saint François d’Assise, Fondateur de l’ordre des Frères mineurs, Patron de l’Italie (✝ 1226)',
```

Reply:
@emagnier, here I need to confirm if the `principal` (patron) word should be removed or not.

Reply:
IMHO you can remove it. In french I see we only specify it when it is 'secondary' or anything other than 'principal'.

Reply:
Okay, I’ll remove `principal`.

Actually, I’ll do the following (in all locales):
- `principal patron(ess)` → `patron(ess)`;
- `secondary patron(ess)` → `copatron(ess)`.

Reply:
@emagnier, I have just noticed that you use `Copatron{s|ne}` in all keys, but not in `joan_of_arc_virgin_secondary_patroness_of_france`. Can I change its value to `Sainte Jeanne d’Arc, Vierge, Copatronne de la France (✝ 1431)`?

Reply:
In the French lectionary (its index) you shared with me, I could find only three patronage titles (I think its missing many of them, e.g. St Joan of Arc; note that the titles are written in lowercase unlike in the romcal codebase):
- `patronne`:
   - `Ste Anne, patronne de la province civile de Québec, et S. Joachim, parents de la Vierge Marie`;
   - `Notre-Dame de Guadaloupé (Mexique), patronne des Amériques`
- `patrons secondaires`:
   - `S. Jean de Brébeuf, S. Isaac Jogues, prêtres, et leurs compagnons Jésuites, martyrs, † de 1642 à 1649, patrons secondaires du Canada`.

If that is kept in the latest edition of the French lectionary (which you told me was to be published soon, probably it is already published), IMHO we should disregard this inconsistency and try to be always consistent and accurate. This would help us when we implement #201. However, I presume that there were others who see this as an error from the consistency point of view.

Reply:
It’s because the patronage titles are generally described in the small presentation available in the missal. It might be the reason not always to be in the title, and probably for simplicity (not to have too long titles). But you are right, it's reasonable to make them more consistent.

FYI the new French missal has been postponed to the next year. But the titles from the lectionary will not change, especially that it's a recent publication (2016). It will just not contain the latest blessed/saints.

Reply:
Thanks for the info, however, you did not answer me how should we localise `copatron(ess)` into French. :smiley:

Reply:
`co-patron` and `co-patronne` is the correct form in French.

Reply:
Okay, @emagnier. As we currently use title-case in French, I think we should abide it and use `Co-Patron(ne|s)`.

Reply:
It will be better to make them all lowercase, like in the missal and lectionary.

Reply:
I agree with you, but then we should change all the keys according to the wording from the French Missal, however, as I said, I leave this job to you. For now (to be consistent with the current wording in the romcal codebase), I suggest to use title-case in French, until you’ll find some time to fix the wording.